src/lib/pick-layers.js

```diff
@@ -29,6 +29,11 @@ export function pickLayers(gl, {
   }, () => {
     let zOrder = 0;
 
+    const blend = gl.getParameter(GL.BLEND);
```

Comment:
I'd keep these three lines tightly together without empty lines, and put the comment as the first of the three lines

Maybe change comment to 
    // disable blending while picking, restore when done